aiida/orm/nodes/data/code/abstract.py

```diff
@@ -283,10 +282,10 @@ def cli_validate_label_uniqueness(_, __, value):
     @classmethod
     def get_cli_options(cls) -> collections.OrderedDict:
         """Return the CLI options that would allow to create an instance of this class."""
-        return collections.OrderedDict(cls._get_cli_options())
+        return collections.OrderedDict(cls.get_cli_options())
 
     @classmethod
-    def _get_cli_options(cls) -> dict:
```

Comment:
This change is incorrect. The idea is that it is private and that subclasses override this. The base class already defines the public `get_cli_options` which should be used by users of this class

Reply:
Ah, I didn't notice that, shame failure. Thanks!